File: algorithms/0633. Sum of Square Numbers.cpp
class Solution
{
public:
    bool judgeSquareSum(long c)
    {
        for (long a = 0; a * a <= c; ++a)
        {
            double b = sqrt(c - a * a);
            
            if (b == (long)b)
            {
                return true;
            }
        }
        
        return false;
    }
};

File: 0633. Sum of Square Numbers.cs
public class Solution
{
    public bool JudgeSquareSum(int c)
    {
        for (int a = 0; a <= Math.Sqrt(c); ++a)
        {
            var b = Math.Sqrt(c - a * a);

            if (b == (int)b)
            {
                return true;
            }
        }

        return false;
    }
}